scripts/bundle-generation/PLATFORM-AWARE-CHANGES.py

```
@@ -0,0 +1,434 @@
+    """
+    values_path = os.path.join(chart_path, "values.yaml")
+
+    if not os.path.exists(values_path):
+        logging.warning(f"No values.yaml found at: {values_path}")
+        return {}
+
+    try:
+        with open(values_path, 'r') as f:
+            values = yaml.safe_load(f)
+            return values if values else {}
+    except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)